feature/floor-map/src/main/java/io/github/droidkaigi/confsched2023/floormap/FloorMapScreenViewModel.kt

```diff
@@ -36,6 +39,14 @@ class FloorMapScreenViewModel @Inject constructor(
             floorLevel = floorLevel,
             floorMapUiState = floorMapUiState,
             floorMapSideEventListUiState = floorMapSideEventListUiState,
+            basementFloorMapSideEventListUiState = FloorMapSideEventListUiState(
+                floorMapAllSideEventListUiState.sideEvents.filter { it.floorLevel == FloorLevel.Basement }
+                    .toImmutableList(),
+            ),
+            groundFloorMapSideEventListUiState = FloorMapSideEventListUiState(
+                floorMapAllSideEventListUiState.sideEvents.filter { it.floorLevel == FloorLevel.Ground }
+                    .toImmutableList(),
+            ),
```

Comment:
@momomomo111 @takahirom 

In mobile layout, event list will change according to the floor level switch. 
large screen should be display 2 floor information, so need each floors event.
I split event list basement and ground, but this change not scalable.

Do you have any good idea?

Reply:
It will be the most efficient way for memory and computation but I think this is a consistent way to build UI.

```
FloorMapScreenUiState
- sealed interface FloorMapContentUiState
  - LargeFloorMapContentUiState
    - BaseListUiState
    - GroundListUiState
  - SmallFloorMapContentUiState
    - ListUiState
```

Reply:
awsome.
I'll try it.

Reply:
If I may ask, is it correct to say that the lack of scalability means that it may not be flexible enough to accommodate an increase or decrease in the number of floors?

Reply:
@momomomo111 
>  it may not be flexible enough to accommodate an increase or decrease in the number of floors?

Yes. 
We don't need to worry about extending the number of floors at this conference?

Reply:
> I think this is a consistent way to build UI.

fixed
https://github.com/DroidKaigi/conference-app-2023/pull/814/commits/f3d61ae149dd853b87ac76f24bf8b5bb61a604bd

Reply:
@bowyer-app 
Thank you for your consideration.
At this time, I believe there is no need to consider expanding the floor.
This is because, with larger screens, significant design changes would be necessary, among other reasons.

I think I'm fine with the changes you made.🙏

Reply:
@momomomo111 
> At this time, I believe there is no need to consider expanding the floor.
This is because, with larger screens, significant design changes would be necessary, among other reasons.

Understood.

> I think I'm fine with the changes you made.

Glad to hear you say that.